hare/algorithm.go

```diff
@@ -658,7 +669,10 @@ func (proc *consensusProcess) beginNotifyRound(ctx context.Context) {
 // passes all pending messages to the inbox of the process so they will be handled.
 func (proc *consensusProcess) handlePending(pending map[string]*Msg) {
 	for _, m := range pending {
-		proc.inbox <- m
+		select {
+		case <-proc.ctx.Done():
```

Comment:
I assume when the context is `Done` the for loop should break here?

Reply:
you assume correctly. thanks